pkg/ccl/backupccl/restore_planning.go

```diff
@@ -1834,7 +1837,7 @@ func doRestorePlan(
 			RevalidateIndexes:  revalidateIndexes,
 			DatabaseModifiers:  databaseModifiers,
 			DebugPauseOn:       debugPauseOn,
-			RestoreSystemUsers: restoreStmt.SystemUsers,
+			RestoreSystemUsers: restoreSystemUsers,
```

Comment:
nit: maybe change to `RestoreSystemUsers: restoreStmt.DescriptorCoverage == tree.SystemUsers` and move your comment above the field?

Reply:
Done.